.github/actions/spelling/patterns.txt

```diff
@@ -1,21 +1,9 @@
-[Ll][Oo][Mm][Bb][Ii][Qq]
 (\((.*?)\))
 (https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})
 [Hh]ome[Pp]age
-[Dd]otnet
-[Ll][Ii][Nn][Qq]
-[Ll]ocalhost
-[Mm][Vv][Cc]
-[Uu]ncomment
-[Jj]ira
-[Ii][Ii][Ss]
-[Cc]mdlet
-[Aa]tata
-[Cc][Ll][Oo][Uu][Dd][Ss][Mm][Ii][Tt][Hh]
-[Ss][Dd][Kk]
-[Rr][Ee][Aa][Dd][Mm][Ee]
-[Oo][Ss][Oo][Cc][Ee]
-[Ss][Ff][Cc]
+[Cc](loud|LOUD)[Ss](mith|MITH)
 [Xx][Uu]nit
-[Xx][Pp][Aa][Tt][Hh]
-[Rr]eadme
+[Xx][Pp]ath
+[{]?[0-9a-zA-Z]{8}-([0-9a-zA-Z]{4}-){3}[0-9a-zA-Z]{12}[}]?
+[s][h][a][0-9]{3}-[0-9a-zA-Z\+\/]{64}
+\b([a-f0-9]{40})\b
```

Comment:
Let's use the same order of digits and letters here as above:
```suggestion
\b([0-9a-f]{40})\b
```
Are uppercase letters not allowed here? If yes, then a comment should mention that.

Reply:
Git commit hash IDs don't contain uppercase letters, so no need for those to be included in this one.

Reply:
Thanks!